Lombiq.Tests.UI/Extensions/EmailUITestContextExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using Atata;
+using Lombiq.Tests.UI.Helpers;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using System;
+
+namespace Lombiq.Tests.UI.Extensions
+{
+    public static class EmailUITestContextExtensions
```

Comment:
I'd now move the rest of the SMTP-related extensions here too.

Reply:
I think that's only `GoToSmtpWebUI`.